packages/auth-construct/src/construct.ts

```diff
@@ -108,32 +121,56 @@ export class AmplifyAuth
     );
 
     // UserPool - Identity Providers
-    const providerSetupResult = this.setupIdentityProviders(
+    this.providerSetupResult = this.setupIdentityProviders(
       this.userPool,
       props.loginWith
     );
-    this.oauthMappings = providerSetupResult.oauthMappings;
 
-    // UserPool Client
+    this.domainPrefix = props.loginWith.externalProviders?.domainPrefix ?? '';
```

Comment:
why do we need `?? ''` if we check it in `if` below ?
is `this.domainPrefix` type correct?

Reply:
good catch, i updated the type